connector/connect/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectStreamHandler.scala

```diff
@@ -50,9 +49,9 @@ class SparkConnectStreamHandler(responseObserver: StreamObserver[Response]) exte
 
   def handlePlan(session: SparkSession, request: proto.Request): Unit = {
```

Comment:
just for curiosity: why do we need to differentiate command and query requests?

Reply:
It is because we have divided it at the proto level: in proto, query and command are different.

The primary reason for dividing is, query runs and return rows. Command apply a state change without return anything (except exceptions probably).

Maybe there is better way to mode all of these. For example, will Catalog API fit into this model? I am still thinking about it but do not have a clear answer now.

Reply:
Commands can return rows, for example, `SHOW TABLES`. Even for DML commands, we may also return rows in the future, to include metrics (num rows inserted, etc.)

Reply:
I see. Yeah probably we can refactor proto to call all as a Plan....

Reply:
@cloud-fan 

From what I can tell in the near term we probably won't touch `Command` proto again. How about let us wait for some time to see if there is any new need that actually needs a separate `Command` proto. If not we can deprecate it and merge with `Relation` proto?

Reply:
SGTM